tests/conftest.py

```diff
@@ -181,10 +194,21 @@ def build_test_backends_list(metafunc):
 
 
 def build_test_backends_list_cfe(metafunc):
+    # adding deprecation warnings
+    if metafunc.config.getoption("--no-postgresql"):
+        warnings.warn(
+            "--no-sqlalchemy is being deprecated in favor of the --postgresql flag. Please adjust your tests accordingly",
```

Comment:
Love this whole PR. How about leveraging our new code standard for deprecation by adding the version in the deprecation warning and in a comment - see more detail here: https://superconductive.atlassian.net/wiki/spaces/SUP/pages/368476281/Code+Standards#Deprecation-Warnings-and-Retiring-Deprecated-APIs-(proposal%2C-policy-to-be-decided)

Reply:
Thank you very much for catching this @anthonyburdi. Added the v0.14 flag